scripts/vgpu-name.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```

Comment:
Please add a license notification that at least contains this:
```
# This file is part of the "vgpu_unlock" project, and is distributed under the MIT License.
# See the LICENSE file for more details.
```
That way it is clear what license that applies to the file and where it comes from if it is ever copied/moved out of the repository.

Reply:
I've added the requested lines :)